vendor/modules.txt

```diff
@@ -50,13 +50,17 @@ github.com/modern-go/concurrent
 github.com/modern-go/reflect2
 # github.com/newrelic/infra-integrations-sdk v1.0.1-0.20201016141521-6fdfe28ad6c0
 github.com/newrelic/infra-integrations-sdk/args
-github.com/newrelic/infra-integrations-sdk/data/errors
-github.com/newrelic/infra-integrations-sdk/data/event
-github.com/newrelic/infra-integrations-sdk/data/inventory
-github.com/newrelic/infra-integrations-sdk/data/metadata
-github.com/newrelic/infra-integrations-sdk/data/metric
-github.com/newrelic/infra-integrations-sdk/integration
-github.com/newrelic/infra-integrations-sdk/log
+# github.com/newrelic/infra-integrations-sdk/v4 v4.0.0
+github.com/newrelic/infra-integrations-sdk/v4/args
+github.com/newrelic/infra-integrations-sdk/v4/data/errors
+github.com/newrelic/infra-integrations-sdk/v4/data/event
+github.com/newrelic/infra-integrations-sdk/v4/data/inventory
+github.com/newrelic/infra-integrations-sdk/v4/data/metadata
+github.com/newrelic/infra-integrations-sdk/v4/data/metric
+github.com/newrelic/infra-integrations-sdk/v4/integration
+github.com/newrelic/infra-integrations-sdk/v4/log
+# github.com/newrelic/infrastructure-agent v0.0.0-20201127092132-00ac7efc0cc6
+github.com/newrelic/infrastructure-agent/pkg/event
```

Comment:
The SDK for some reasons depends on agent code, I asked if there is any reason for that :
https://github.com/newrelic/infra-integrations-sdk/pull/252